.gitignore

```diff
@@ -57,3 +57,12 @@ target/
 
 .ipynb_checkpoints/
 .python-version
+
+#Pycharm settings
```

Comment:
Already removed these lines in latest commits, should be clean by now. I messed up the commit log a little bit. Just check the latest.